dao/pom.xml

```diff
@@ -8,13 +8,18 @@
         <version>1.0.0</version>
     </parent>
     <modelVersion>4.0.0</modelVersion>
-
     <artifactId>dao</artifactId>
+    <packaging>jar</packaging>
```

Comment:
you have an empty file `base.sql`, what is it for?

Reply:
The file base.sql deleted. 
@valevaty98 done